index.html

```diff
@@ -129,55 +129,412 @@ <h2 data-dft-for="">
         "https://wicg.io">WICG</a>.
       </p>
     </section>
-    <section>
+    <section data-cite="DOM">
       <h2>
-        <code><dfn>note_taking</dfn></code> member
+        Installation prompts
       </h2>
       <p>
-        The <code>note_taking</code> member of the <a data-cite=
-        "appmanifest#web-application-manifest">Web Application Manifest</a> is
-        an <a data-cite="appmanifest#dfn-object">object</a> that contains
-        information related to note-taking. It has the following members:
+        There are multiple ways that the installation process can be triggered:
       </p>
       <ul>
-        <li>[=note_taking/new_note_url=]
+        <li>An end-user can <dfn data-lt="manual installation">manually</dfn>
+        trigger the installation process through the user agent's
+          <abbr title="User Interface">UI</abbr>, directly invoking the steps
+          to <a>present an install prompt</a>.
+        </li>
+        <li>The installation process can occur through a <dfn>site-triggered
+        install prompt</dfn>: the site can programmatically request that the
+        user agent present an install prompt to the user. The user agent MAY
+        restrict the availability of this feature to cases where, for instance,
+        there are sufficient <a data-cite="appmanifest#installability-signals">
+          installability signals</a> to warrant <a data-cite=
+          "appmanifest#dfn-installed">installation</a> of the web application.
         </li>
       </ul>
+      <p>
+        In any case, the user agent MUST NOT <a>present an install prompt</a>
+        if the document is not <a data-cite=
+        "appmanifest#dfn-is-installable">installable</a>.
+      </p>
+      <p>
+        The user agent MAY, at any time (only if the document is <a data-cite=
+        "appmanifest#dfn-is-installable">installable</a>), run the <a>steps to
+        notify that an install prompt is available</a> at any time, giving the
+        site the opportunity to show a <a>site-triggered install prompt</a>
+        without the user needing to interact with the user agent UI.
+      </p>
+      <p>
+        To <dfn data-lt=
+        "presenting an install prompt|presentation of the install prompt">present
+        an install prompt</dfn>:
+      </p>
+      <ol>
+        <li>Show some user-agent-specific UI, asking the user whether to
+        proceed with installing the app. See <a data-cite=
+        "appmanifest#installation-sec">privacy and security considerations</a>
+        for recommendations relating to this UI. The <var>result</var> of this
+        choice is either {{AppBannerPromptOutcome|accepted}}
+          <a data-link-for="AppBannerPromptOutcome">accepted</a> or
+          <a data-link-for="AppBannerPromptOutcome">dismissed</a>.
```

Comment:
```suggestion
          {{AppBannerPromptOutcome\accepted}} or
          {{AppBannerPromptOutcome\dismissed}}.
```

Same globally to the rest of them.